.pre-commit-config.yaml

```diff
@@ -26,3 +26,10 @@ repos:
     rev: 6.1.0
     hooks:
     -   id: flake8
+-   repo: https://github.com/codespell-project/codespell
+    rev: v2.1.0
+    hooks:
+    -   id: codespell
+        name: 'Spell checker'
+        entry: codespell
+        language: python
```

Comment:
codespell docs says,
> If one configures codespell using the pyproject.toml file instead use:
```
- repo: https://github.com/codespell-project/codespell
  rev: v2.2.4
  hooks:
  - id: codespell
    additional_dependencies:
      - tomli
```

so, pls add additional_dependencies.
and you can clear name, entry, language.